spell-checker/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <script>
+    function spellChecker(userWords, correctWords) {
+      const correctArr = [];
+      const wrongArr = [];
+      for (let i = 0; i < userWords.length; i++) {
+        if (userWords[i] === correctWords[i]) {
+          correctArr.push(correctWords[i])
+        } else {
+          wrongArr.push(userWords[i])
+         
+        }
+      }
+      return `correct: ${correctArr}, incorrect: ${wrongArr}`
```

Comment:
Rather than returning a string, this function should return the object you built.